### ISSUE TITLE
Framework AMD fingerprint scanner not working

### ISSUE THREAD

**Issue by asosnovsky** [NixOS/nixos-hardware#840]
Basically I followed the instructions in https://github.com/NixOS/nixos-hardware/tree/master/framework/13-inch/7040-amd#getting-the-fingerprint-sensor-to-work and it did not work.

I suspect that the firmware installation failed at one point and that the hardware is in some strange corrupt state.

Here is the output for attempting to install it 

```
[root@fwbook:/home/ari/nixos-setup/shells/framework-amd-button-fix]# fwupdtool install --allow-reinstall --allow-older goodix-moc-609c-v01000330.cab
/nix/store/1n2l5law9g3b77hcfyp50vrhhssbrj5g-glibc-2.37-8/lib/libc.so.6: version `GLIBC_2.38' not found (required by /nix/store/az60ii5827bsiyla9dfq5sprkaibarmx-libproxy-0.4.18/lib/libproxy.so.1)
Failed to load module: /nix/store/zd4zpqb3af9zgx7qjkvkmv87xyd2wgph-glib-networking-2.78.0/lib/gio/modules/libgiolibproxy.so
Loading…                 [ -                                     ]/nix/store/1n2l5law9g3b77hcfyp50vrhhssbrj5g-glibc-2.37-8/lib/libc.so.6: version `GLIBC_2.38' not found (required by /nix/store/h5f78zrxgn2rsqqnqd798fzkx23yp4ax-gvfs-1.52.2/lib/gio/modules/libgvfsdbus.so)
Failed to load module: /nix/store/h5f78zrxgn2rsqqnqd798fzkx23yp4ax-gvfs-1.52.2/lib/gio/modules/libgvfsdbus.so
Loading…                 [                                       ]21:08:53.594 FuHistory            schema version 9 is unknown
Waiting…                 [***************************************]
```

Note the `version `GLIBC_2.38' not found` I think this might be the issue, but I have not been able to resolve it.

I have posted more about the journey behind my issues here https://community.frame.work/t/nixos-upgrade-to-amd-motherboard-fingerprint-not-found/43458/2 (there are some outputs there for other debugging that I did)

**Comment by Luflosi** [comment]
I just pretty much followed https://knowledgebase.frame.work/en_us/updating-fingerprint-reader-firmware-on-linux-for-13th-gen-and-amd-ryzen-7040-series-laptops-HJrvxv_za and now my fingerprint scanner works.
TLDR:
- `fwupdmgr enable-remote lvfs-testing`
- `fwupdmgr get-devices 1e8c8470-a49c-571a-82fd-19c9fa32b8c3`
- `fwupdmgr update 1e8c8470-a49c-571a-82fd-19c9fa32b8c3`

However, I don't know how to recover a corrupted fingerprint reader.

**Comment by Redhawk18** [comment]
I have a 16 and it works flawlessly
